jest.config.js

```diff
@@ -0,0 +1,29 @@
+/** @type {import('jest').Config} */
+module.exports = {
+	coveragePathIgnorePatterns: [
+		'/node_modules/',
+		'dist',
+		'lib',
+		'lib-esm',
+		'__tests__',
```

Comment:
lib* is no longer needed as they are moved under `dist`
```suggestion
		'__tests__',
```

Reply:
Nice